### DISCUSSION TITLE
Stabilizer loop rate off warning after low battery (red light on) CF 2.1

### DISCUSSION THREAD

**Discussion by Vrinda73** [bitcraze/discussions#512] [Q&A] [answered]
Hello !
I am facing a weird issue. Here is my story. I was testing yesterday. The battery was critically low after a test. So I went to charge and from then the drone receives thrust command and the propellers run, but it won't take off even after giving full thrust. I did the propeller test and I get the stabilizer loop rate off warning. 
I performed the following steps with no luck:
1. reinstalled cflib
2. updated firmware
3. changed propellers
I tried looking into [https://github.com/bitcraze/crazyflie-firmware/issues/626](url) but did not understand what to do.

Output of my console:

HEALTH: Acc noise floor variance X+Y:0.003606, (Z:0.002227)
HEALTH: Motor M1 variance X+Y: 2.90 (Z:19.74), voltage sag:0.34
HEALTH: Motor M2 variance X+Y: 0.65 (Z:1.45), voltage sag:0.34
HEALTH: Motor M3 variance X+Y: 5.81 (Z:20.50), voltage sag:0.33
HEALTH: Motor M4 variance X+Y: 0.22 (Z:0.24), voltage sag:0.14
HEALTH: Propeller test on M1 [FAIL]. low: 0.0, high: 2.50, measured: 2.90
HEALTH: Propeller test on M3 [FAIL]. low: 0.0, high: 2.50, measured: 5.81
STAB: WARNING: stabilizer loop rate is off (87)

Please help, I have a project deadline coming soon. I would really appreciate if anyone can guide me regarding this.
Thank you so much!

**Comment by krichardsson** [comment, marked as answer]
Hi,
when running the propeller test, the rest of the firmware is still running in the background. The propeller test pretty much hogs the CPU while running the test and the rest of the firmware may issue some warnings. You can safely ignore it when running the propeller test.

**Reply by Vrinda73** [reply]
Thank you very much Kristoffer! That was helpful.
